### ISSUE TITLE
Add web developer friendly links to MDN

### ISSUE THREAD

**Issue by captainbrosset** [web-platform-dx/web-features-explorer#32]
**Help wanted: help me complete the mapping between web-features and MDN URLs**

The features listed in the web-features repository don't come with MDN URLs.
However, linking web-features to MDN URLs is useful for several things:

* If the mapping is available somewhere, then consumers of web-features can use the mapping to link the web-features they display to MDN.
  * One example is the State Of surveys, which use web-features, and which also show MDN URLs. This mapping with help there.
  * Another example is this very site: https://web-platform-dx.github.io/web-features-explorer/, which shows MDN links on individual feature pages. This mapping helps show the right links.
* The other way around is true too. For MDN, which shows baseline badges, having a one-to-one mapping from MDN pages to web-features can help find the right web-features.

The mapping is, for now, maintained in this file: https://github.com/web-platform-dx/web-features-explorer/blob/main/mdnDocsOverrides.json

There are still a couple hundred features left to be mapped. Here's how the mapping works:

```json
{
  "<web-feature-id>": [
    "<MDN-page-slug>"
  ]
}
```

Each feature points to one or more MDN URL. Each MDN URL is partial, only the slug is needed, not the root URL. Here's an example:

```json
{
  "aborting": [
    "Web/API/AbortController",
    "Web/API/AbortSignal"
  ]
}
```

The root URL is `https://developer.mozilla.org/en-US/docs/`, so the two slugs above end up pointing to: https://developer.mozilla.org/en-US/docs/Web/API/AbortController and https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal.

**Comment by jsnkuhn** [comment]
If this is really as straightforward as it seems I'll give this some time. Searching for and then copy/pasting links into the .json is all you are really looking for here?

**Comment by captainbrosset** [comment]
Pretty much, yes. It takes a little bit of reading of the feature description first, to make sure you know what the feature is about. And it takes a little bit of figuring out whether we want just one link to MDN (which is the most common case), or several (e.g. when a feature is about, say, two different methods that are related, but that are not described by just one MDN page).

Thank you for helping !

**Comment by captainbrosset** [comment]
The object keys in that file correspond to IDs of features defined in the web-features repository. They should not be changed. 

To find out more about a feature, given its ID, do either of these things:

- Go to https://github.com/web-platform-dx/web-features and look for the file in the `features` folder that has the same name as the feature ID. For example, borders.yml. 
- Or open the feature on the explorer website: https://web-platform-dx.github.io/web-features-explorer/features/borders (replace the last part with your feature ID).

For this project, you should consider the feature IDs as fixed and should not change them in the mdnDocsOverrides.json file. The idea is, for each feature listed in there to find the MDN page that best describes the feature. 

Does that help ?

**Comment by jsnkuhn** [comment]
Yes, thank you that helps. Had a go at A-C this afternoon. Have a look and let me know if I'm on the right track before I send a pull:

https://github.com/web-platform-dx/web-features-explorer/compare/main...jsnkuhn:web-features-explorer:main 

Do you want them all at once or is it fine in sections like this one?

**Comment by captainbrosset** [comment]
@jsnkuhn looks like a great start. Please do send a pull request. And, yes, in sections like this one is best. It'll make it easier to review.

**Comment by jsnkuhn** [comment]
Dropping this in here as a reminder to jump ahead to the things listed on "CSS Wrapped 2024" that need MDN links before going back to working alphabetically. Will watch out for anything similar from the other vendors:

https://chrome.dev/css-wrapped-2024/

**Comment by captainbrosset** [comment]
It's coming apparently : https://github.com/mdn/content/pull/37118

**Comment by captainbrosset** [comment]
Headsup that the docs file has moved to https://github.com/web-platform-dx/web-features-explorer/blob/main/additional-data/mdn-docs.json.

**Comment by jsnkuhn** [comment]
Ran into another CSS property that seems to be oddly missing an MDN article: `object-view-box`. Been in Chromium since Aug 2022 but can't find an article?

**Comment by captainbrosset** [comment]
> Ran into another CSS property that seems to be oddly missing an MDN article: `object-view-box`. Been in Chromium since Aug 2022 but can't find an article?

I filed https://github.com/mdn/mdn/issues/612 for this.

**Comment by jsnkuhn** [comment]
Alright, made it through the list once. Skipped things either without MDN articles or things I wasn't familiar enough with to know if I was linking to the right article.

**Comment by captainbrosset** [comment]
Thank you @jsnkuhn, great work on this, much appreciated. 

I will review your latest PR as soon as I can (likely on one or two days from now).

**Comment by jsnkuhn** [comment]
The `::details-content` article is up:
https://developer.mozilla.org/en-US/docs/Web/CSS/::details-content

(Seemingly no mention of `::details-content` in the `<details>` article though?)

Doesn't seem worth making a PR for one link?

**Comment by captainbrosset** [comment]
Thanks. I just added it to the file.
